packages/fast-components-class-name-contracts-base/src/Slider/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * The class name contract for the slider component
```

Comment:
I'm getting naming errors bootstraping - can you ensure this gets checked in under the `slider` directory (note the lower case)

Reply:
Know any tricks to force git to rename it?  I've changed the folder to all lower case and force pushed but the server version seems to retain the capital S.

Reply:
`git mv Slider slider-tmp` then `git mv slider-tmp slider`

Reply:
Thanks, that worked